.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+---
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: Build a Docker image
+    timeout-minutes: 30
+    if: github.event_name == 'push'
```

Comment:
That line can be removed together with `pull_request:` above

Reply:
Done